packages/mui-joy/src/Menu/Menu.tsx

```diff
@@ -153,6 +153,26 @@ const Menu = React.forwardRef(function Menu(inProps, ref: React.ForwardedRef<HTM
     [modifiersProp],
   );
 
+  if (anchorEl) {
```

Comment:
Let's wrap this warning in development
```suggestion
  if (process.env.NODE_END === 'development') {
    
  }
```